cloud/amazon/ec2.py

```diff
@@ -1304,7 +1304,9 @@ def startstop_instances(module, ec2, instance_ids, state, instance_tags):
                 except EC2ResponseError, e:
                     module.fail_json(msg='Unable to change state for instance {0}, error: {1}'.format(inst.id, e))
                 changed = True
+            existing_instances_array.append(inst.id)
 
+    instance_ids = list(set(existing_instances_array + (instance_ids or [])))
```

Comment:
Instance_ids is part of the return value from the module.  It isn't going to mess up users that this has changed?